config/parser.go

```diff
@@ -33,14 +33,14 @@ func getStageTypeModuleName(stageType string) string {
 	return strings.ToLower(stageType)
 }
 
-func Parse(configData *map[interface{}]interface{}) (*pipelines.Pipeline, error) {
+func Parse(configData *map[interface{}]interface{}) (*pipelines.Resources, error) {
 	envs := NewEnvVariables()
 	return ParseWithSpecifiedEnvs(configData, envs)
 }
 
 // TODO: need refactoring
```

Comment:
Did you finished refactoring?
If you didn't , please add comment what do you refactor about.